publish/index.js

```diff
@@ -37,7 +37,13 @@ export const publish = async ({
       provider_id,
       cid,
       provider_address,
-      protocol
+      protocol,
+      alternative_provider_check_status_code,
+      alternative_provider_check_timeout,
+      alternative_provider_check_car_too_large,
+      alternative_provider_check_end_at,
+      alternative_provider_check_protocol,
+      alternative_provider_check_provider_id
```

Comment:
Do we have any concerns about the size of JSON measurements uploaded to Storacha? The prefix `alternative_provider_check_` has 28 characters, it's repeated 6x per measurement, and we have ~200K measurements/round - that's 33MB/round to store the prefix.

Have you considered re-building the nested `alternative_provider_check` object instead?

Example measurement to show what I mean:

```json5
{
  "miner": "f01...",
  "provider_id": "somePeerId",
  // ...
  "alternative_provider_check": {
    "provider_id": "someOtherPeerId",
    // ..
  }
}
```

Maybe this is not a concern right now? Let's discuss!

Reply:
I haven't been thinking about the size itself. What started as adding single field to the measurement structure quickly turned into adding more and more of them.

I like your suggestion 👍🏻 

Another alternative could be to store a alternative provider check as a separate measurement  with an additional flag. WDYT?